src/rt/util/hash.d

```diff
@@ -95,15 +90,18 @@ size_t hashOf( const(void)* buf, size_t len, size_t seed )
     return hash;
 }
 
-// Check that hashOf works with CTFE
-@nogc nothrow pure unittest
```

Comment:
OK, but I don't see why the printf is necessary (and it can be put under `debug` so nothing is lost). Anyway can you just leave a link to bugzilla issue for the CTFE problem?